tasks/hacbs-test-evaluation.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: hacbs-test-evaluation
+spec:
+  description: >-
+    Show summary of HACBS tests
+  workspaces:
+    - name: workspace
+  params:
+    - name: BASE_IMAGE
+  steps:
+    - name: summary  
+      image: quay.io/redhat-appstudio/hacbs-test:stable
+      workingDir: $(workspaces.workspace.path)/hacbs
+      script: |
+        #!/usr/bin/env bash
+        EXIT=0
+
+        # sanity-label-check
+        FAILURES=$(jq '.[] | .failures // {} | .[] | .msg' sanity-label-check/sanity_label_check_output.json)
+        if [ -n "$FAILURES" ]; then
+          echo sanity-label-check test FAIL:
+          echo "$FAILURES"
+          EXIT=1
```

Comment:
We don't want to fail the pipeline if any of the test tasks fail - the intent is for the tasks to only report the outcome of each individual test in the `test_name_output.json` file and allow EC to make the decision to release or block the component.